operator/internal/manifests/internal/config/loki-runtime-config.yaml

```
@@ -57,13 +57,10 @@ overrides:
     {{- with $l.Blocked }}
     blocked_queries:
     {{- range $blockedQuery := . }}
-    - {{ with $blockedQuery.Pattern -}}
-      pattern: |
-        {{ . | yamlBlock "        "  }}
+    - pattern: {{ $blockedQuery.Pattern }}
+      {{ with $blockedQuery.Regex -}}
+        pattern: {{ . | printf "%q" }}
```

If I'm not mistaken, I think here is regex instead of `pattern` 
```suggestion
    - pattern: {{ $blockedQuery.Pattern }}
      {{ with $blockedQuery.Regex -}}
       regex: {{ . | printf "%q" }}
```

---

Yes, `pattern` is duplicated now, the template also needs a change and there's a slight change to the "zero-length behaviour" of the templates:

```suggestion
    - pattern: {{ $blockedQuery.Pattern }}
      {{- with $blockedQuery.Regex }}
      regex: {{ . }}
```

Don't forget to also change to `{{- end }}` on the next line (not part of the highlight, so can't adjust the suggesstion).

---

You're right. I quickly pushed this change yesterday without even testing and now the results are messed up ^^"
It's also not what you suggested though.. There are some use cases when neither `pattern` nor `regex` are used. So I'll update the template to take this into consideration..

---

> It's also not what you suggested though..

Sorry for the confusion. Usually I just provide a code suggestion to avoid misunderstandings.